lib/ddtrace/propagation/distributed_headers.rb

```diff
@@ -44,5 +40,11 @@ def header(name)
 
       @env[rack_header]
     end
+
+    def id(header)
+      value = header(header).to_i
+      return if value == 0 || value >= Span::MAX_ID
```

Comment:
I assumed that `0` is truly invalid as an ID, but definitely don’t know for sure.

Reply:
Btw, is there a trace specification available?

Reply:
At this moment documentation is not available. But since this issue surfaced its something on our radar.